package.json

```diff
@@ -83,7 +82,6 @@
     "jquery-creditcardvalidator": "https://github.com/PawelDecowski/jquery-creditcardvalidator/archive/v1.1.0.tar.gz",
     "jquery-panelslider": "https://github.com/eduardomb/jquery-panelslider/archive/1.0.0.tar.gz",
     "summernote-nugget": "https://github.com/pHAlkaline/summernote-plugins",
-    "literallycanvas": "https://github.com/literallycanvas/literallycanvas/archive/v0.4.14.tar.gz",
```

Comment:
I'd watch out here; as I recall, this didn't work for me in the past. Copied this from my notes:
"
TODO had to keep the literallycanvas/react duo since literallycanvas is basically a dead project and using the most recent working distribution. Goal will be to migrate to another tool (and could then drop react).
"

Reply:
Didn't get to test this yet so yeah. I'll figure it out

Reply:
As I mention when I brought in 4.14 it works with legacy.
I believe >4.14 react version was bumped and does not work though I didn't try to make work.

Reply:
@tywrenn, I wouldn't wast your time on trying to test this. I spent significant effort on trying to get to most recent version and also on the associated react. The npm package is basically missing a critical file. I'm going to make an issue to replace literallycanvas with another tool; this could be a good educational initial project (moderate complexity and short/medium term) for a GSoC student to work on.